### PR TITLE
fix/single-char-sub-sup

### PR DESCRIPTION
**Pull request by builtbywill**

### CODE REVIEW

**Review by caseyveronica4 — APPROVED** [review]

File: latex2sympy.py [hunk 1]
```diff
@@ -68,6 +72,23 @@ def process_sympy(sympy, variable_values={}):
...
+    '''
+
+    # find any single char sup/sub and wrap them in "{}"
+    # e.g. `4^26^2` => `4^{2}6^{2}`
+    # e.g. `x_22` => `x_{2}2`
+
+    # NOTE: this does modify variable names, but see `atom.VARIABLE()` processing below where this is reverted.
+    # this appears to be the easiest way without making the regex here too complex
+
+    pre_processed_latex = re.sub(r'([\^_])([0-9a-zA-Z])', '\\1{\\2}', latex)
```
Comment (jkdufair) [inline latex2sympy.py:87]: Not super familiar with this codebase. Is there any chance the latex would have  characters outside of `0-9a-zA-Z` that we'd need to account for here at all?

Comment (builtbywill) [reply latex2sympy.py:87]: @jkdufair I don't think so. This is specifically for when the sub/sup is a single character, so any latex commands would already be wrapped as they are always more than one char.

**Review by jkdufair — APPROVED** [review]

**Review by tragessere — APPROVED** [review]

File: tests/context.py [hunk 2]
```diff
@@ -135,7 +135,7 @@ def get_gcd_lcm_examples(cmd, func, func_eval, multi_func):
         (f"{cmd}(6128, 24)", func(6128, 24)),
         (f"{cmd}(24, 6128)", func(24, 6128)),
         (f"{cmd}(1E20, 1000000)", func(Rational('1E20'), 1000000)),
-        (f"{cmd}(128*10^32, 1)", func(Rational('128E32'), 1)),
+        (f"{cmd}(128*10^{{32}}, 1)", func(Rational('128E32'), 1)),
```
Comment (lily514) [inline tests/context.py:138]: @builtbywill why does this need to be in double brackets? shouldn't {32} be enough? :confused:

Comment (builtbywill) [reply tests/context.py:138]: @lily514 because the "f" makes it a string format, so the "cmd" can be inserted, and we need to escape the braces inside

**Review by lily514 — APPROVED** [review]

**Pull request merged**